### PR TITLE
created avatar-button components

### PR DESCRIPTION
**Pull request by agrawalshanu708**

### CODE REVIEW

File: avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (zabihshaik) [inline avatar/avatar.html:7]: Better to add meta tag with your name

File: avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  
+        <!-- -----------------------------size-avatar -->
+
+        <h2>Image Avatar</h2>
+        <p>
+          Image avatars can be created by passing standard img props src or srcSet
+          to the component.</p>
+          <div class="avatar-container">
+          <img
+            class="img-av"
```
Comment (zabihshaik) [inline avatar/avatar.html:58]: Better to use an understandable class name, like image-avatar

**Review by tanishk15sharma — APPROVED** [review]

File: button/button.html [hunk 3]
```diff
@@ -0,0 +1,243 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BUTTON</title>
```
Comment (zabihshaik) [inline button/button.html:7]: Better to add a meta tag with your name

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just check the comments please.

**Pull request merged**